Add X,y,Y to ignored list (#69)
scikit-learn/enhancement_proposals: slep006/proposal.rst

the time of writing this proposal, all metadata requested in the core library
are sample aligned.

> Also note that ``X``, ``y``, and ``Y`` input arguments are never automatically

Reviewer: This is the only mention of upper-case `Y` in this document. (I suspect it is for `PLS`) Do you think this will be confusing?

Author: I mean, from the perspective of the SLEP and the implementation, we don't really care why it's here. These arguments never enter the routing scheme.

Reviewer 2: Small interjection to bring information: `Y` is also used as an argument for some [`pairwise_distances`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.pairwise_distances.html)-based APIs.

Reviewer: In case of `pairwise_distance`, `Y` is a feature array. In the context of this SLEP, `X` and `y` are feature and target data. I am okay with leaving this statement as is.

> added to the routing mechanism and they are routed and passed around the way
> they have been passed around until now.

Reviewer: For those that do not know what the previous behavior:

```suggestion
added to the routing mechanism and are always passed into their
respective methods.
```
